source/_integrations/sensoterra.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Sensoterra
+description: Instructions on how to integrate your Sensoterra probes with Home Assistant.
+ha_release: 2024.7
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Poll
+ha_config_flow: true
+ha_domain: sensoterra
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@markruys'
+ha_integration_type: integration
+ha_quality_scale: silver
+---
+
+This {% term integration %} communicates with your [Sensoterra](https://sensoterra.com) soil moisture probes using the public [Sensoterra Customer API](https://monitor.sensoterra.com/api/v3/) via HTTPS.
+
+## Prerequisites
+
+Soil moisture probes can be purchased at Sensoterra.com. Create a free account in the Sensoterra app and register your probes. The Sensoterra Home Assistant integration needs your credentials to obtain a token to access your probe data.
+
+## Sensors
+
+The integration provides the following sensors for each probe:
+
+- Soil moisture (either in volumetric-% or in SI)
+- Temperature
```

Comment:
What is the unit used here? degrees Celsius

Reply:
Drafting this PR as there is a review comment. Please undraft it again, once the PR is ready.